Mathlib/Algebra/Central/Basic.lean

```diff
@@ -53,4 +54,18 @@ lemma baseField_essentially_unique
   obtain ⟨x', H⟩ := H
   exact ⟨x', (algebraMap K D).injective <| by simp [← H, algebraMap_eq_smul_one]⟩
 
+lemma of_algEquiv (e : D ≃ₐ[K] D') : IsCentral K D' where
+  out x hx :=
+    have ⟨k, hk⟩ := h.1 ((MulEquivClass.apply_mem_center_iff e.symm).mpr hx)
+    ⟨k, by simpa [ofId] using congr(e $hk)⟩
+
+open MulOpposite in
+/-- This instance combined with the coming `IsSimpleRing` instance for a
+  central simple algebra will be an inverse for an element in `BrauerGroup`,
+  find out more about this in `Mathlib.Algebra.BrauerGroup.Basic`. -/
```

Comment:
This feels like a comment in the code, rather than a docstring. If you want to keep it in the docstring, at least add an actual sentence saying what this function does (right now you just talk about other things you'll be using it to do, rather than what it is).